packages/cli/src/cmds/validator/signers/index.ts

```diff
@@ -106,7 +106,7 @@ export async function getSignersFromArgs(
 
   // Remote keys are declared manually or will be fetched from external signer
   else if (args["externalSigner.pubkeys"] || args["externalSigner.fetch"]) {
-    return getRemoteSigners(args);
+    return getRemoteSigners(args, logger as Logger);
```

Comment:
This type cast is not ok, we allow to pass `console` as logger to `getSignersFromArgs`, for example in voluntary exit command which is also supposed to work with remote signers

https://github.com/ChainSafe/lodestar/blob/ba57d4b655bd229b7271504b518f6ad3ba9a0b23/packages/cli/src/cmds/validator/voluntaryExit.ts#L90

If we try to call a method that does not exist we will get unexpected errors. 

I see a few solutions to this
- restrict logger type to minimal set, remove type cast
- extend passed logger with no-ops if methods do not exist
- remove logging, errors should be logged upstream already

Reply:
I chose the first solution as I find it helpful to have a logger in the externalSignerClient for debugging purposes, I hope I have understood the suggestion correctly.